src/components/Slider/components/Thumb.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import {
+  GestureEvent,
+  PanGestureHandler,
+  PanGestureHandlerEventPayload,
+  TextInput,
+} from 'react-native-gesture-handler'
+import styled from 'styled-components/native'
+import Animated, {useAnimatedProps, useAnimatedStyle} from 'react-native-reanimated'
+import type {Position, Size, TextAlign, ThumbContainerStyle} from '../Slider'
+import {isIOS, metrics} from '../../../helpers/metrics'
+import type {StyleProp, TextProps, ViewStyle} from 'react-native'
+import type {ITheme} from '../../../theme'
+
+const AnimatedText = Animated.createAnimatedComponent(TextInput)
+
+interface ThumbProps {
+  text: string
+  bgColorLabelView?: string
+  labelStyle: StyleProp<TextProps>
+  thumbSize: Size
+  thumbStyle: StyleProp<ViewStyle>
+  thumbComponent?: React.ReactElement
+  alwaysShowValue?: boolean
+  animatedThumbStyle: ReturnType<typeof useAnimatedStyle>
+  opacityStyle: ReturnType<typeof useAnimatedStyle>
+  animatedProps: ReturnType<typeof useAnimatedProps>
+  onGestureEvent: (event: GestureEvent<PanGestureHandlerEventPayload>) => void
+}
+
+const Thumb: React.FunctionComponent<ThumbProps> = ({
+  text,
+  bgColorLabelView,
+  labelStyle,
+  alwaysShowValue,
+  thumbSize,
+  thumbComponent,
+  animatedProps,
+  thumbStyle,
+  animatedThumbStyle,
+  opacityStyle,
+  onGestureEvent,
+}) => (
+  <PanGestureHandler onGestureEvent={onGestureEvent}>
+    <ThumbContainer
+      thumbSize={thumbSize}
+      hasThumbComponent={!!thumbComponent}
+      style={[thumbStyle, animatedThumbStyle]}>
+      <LabelContainer
+        background={bgColorLabelView}
+        style={!alwaysShowValue && opacityStyle}
+        thumbSize={thumbSize}>
+        <TriangleDown background={bgColorLabelView} style={{transform: [{rotate: '180deg'}]}} />
```

Comment:
style={{transform: [{rotate: '180deg'}]}}

why used inline style and not move in here instead

![image](https://user-images.githubusercontent.com/124117862/233567109-c0119bfb-73ba-46d8-9598-1a1b3939dd43.png)

Reply:
updated